packages/btc-wallet/connectors/unisat.ts

```diff
@@ -1,67 +1,75 @@
-import { ConnectorGroup, WalletConnector } from './types'
+import { Unisat } from '../unisat'
 
-const isInstalled = () => window?.unisat !== undefined
+import { type ConnectorGroup, type WalletConnector } from './types'
+
+// See https://github.com/unisat-wallet/unisat-dev-docs/blob/master/wallet-api/api-docs/browser-detection.md
+const provider = (typeof window !== 'undefined' &&
+  (window.unisat_wallet || window.unisat)) as Unisat
+
+// Some wallets (e.g., Binance and OKX) inject similar APIs but are not UniSat.
+// Exclude them from UniSat detection to avoid false positives.
+const isInstalled = () =>
+  !!provider && !(provider.isBinance || provider.isOkxWallet)
 
 const assertInstalled = function () {
   if (!isInstalled()) {
-    throw new Error('Unisat is not installed')
+    throw new Error('UniSat Wallet is not installed')
   }
 }
 
-const unisatWalletConnector = {
+const wallet = {
   async connect() {
     assertInstalled()
     // in order to connect to unisat, we just need to request accounts and the user
     // will be prompted to connect
-    await window.unisat.requestAccounts()
+    await provider.requestAccounts()
   },
   disconnect() {
     assertInstalled()
-    return window.unisat.disconnect()
+    return provider.disconnect()
   },
   getAccounts() {
     assertInstalled()
-    return window.unisat.getAccounts()
+    return provider.getAccounts()
   },
   getBalance() {
     assertInstalled()
-    return window.unisat.getBalance()
+    return provider.getBalance()
   },
   getNetwork() {
     assertInstalled()
-    return window.unisat.getNetwork()
+    return provider.getNetwork()
   },
   id: 'unisat',
   isInstalled,
-  name: 'Unisat',
+  name: 'UniSat',
   onAccountsChanged(handler) {
     assertInstalled()
-    window.unisat.on('accountsChanged', handler)
-    return () => window.unisat.removeListener('accountsChanged', handler)
+    provider.on('accountsChanged', handler)
+    return () => provider.removeListener('accountsChanged', handler)
   },
   onChainChanged(handler) {
     assertInstalled()
     // This event is not listed in the docs, but "networkChanged" doesn't fire
     // See https://github.com/unisat-wallet/extension/issues/211#issuecomment-2290557037
-    window.unisat.on('chainChanged', handler)
-    return () => window.unisat.removeListener('chainChanged', handler)
+    provider.on('chainChanged', handler)
+    return () => provider.removeListener('chainChanged', handler)
   },
   sendBitcoin(toAddress, satoshis, options) {
     assertInstalled()
-    return window.unisat.sendBitcoin(toAddress, satoshis, options)
+    return provider.sendBitcoin(toAddress, satoshis, options)
   },
   switchNetwork(network) {
     assertInstalled()
-    return window.unisat.switchNetwork(network)
+    return provider.switchNetwork(network)
   },
 } satisfies WalletConnector
 
 export const unisat = {
   downloadUrls: {
-    android: 'https://play.google.com/store/apps/details?id=io.unisat',
```

Comment:
Was the Android one removed for any particular reason? I think we should also add the iOS one.